src/braket/circuits/noise_helpers.py

```diff
@@ -210,25 +213,23 @@ def _apply_noise_to_gates_helper(
         noise_index: The number of noise channels applied to the gate
         noise_applied: Whether noise is applied or not
     """
-
     for noise_channel in noise:
         if noise_channel.qubit_count == 1:
             for qubit in intersection:
                 # apply noise to the qubit if it is in target_qubits
                 noise_index += 1
                 new_noise_instruction.append((Instruction(noise_channel, qubit), noise_index))
                 noise_applied = True
-        else:
-            # only apply noise to the gates that have the same qubit_count as the noise.
-            if (
-                instruction.operator.qubit_count == noise_channel.qubit_count
-                and instruction.target.issubset(target_qubits)
-            ):
-                noise_index += 1
-                new_noise_instruction.append(
-                    (Instruction(noise_channel, instruction.target), noise_index)
-                )
-                noise_applied = True
+        # only apply noise to the gates that have the same qubit_count as the noise.
```

Comment:
It's not clear that this doc applies to the elseif, which is probably why the else was split. Someone could easily add a line of code here and confuse everything.

Reply:
This issue is present whether it is split or not. If a line of code is added in either case, the comment should be reevaluated.

Reply:
Thinking more on this, Python comments usually are applicable to the next line/based on indentation. The other option is to add this inline to elif but based on Python convention this is clear the scope of where this comment is applied.